packages/jazz-react-core/src/hooks.ts

```diff
@@ -57,6 +57,7 @@ export function useCoState<V extends CoValue, D>(
   >(
     React.useCallback(
       (callback) => {
+        observable.reset();
```

Comment:
We can also remove this reset, as in theory we do this already with the sync resolution